datasets/us/ofac/enforcement_actions/crawler.py

```diff
@@ -0,0 +1,291 @@
+from datetime import date
+from lxml.html import HtmlElement, fromstring, tostring
+from pydantic import BaseModel, Field
+from rigour.mime.types import HTML
+from typing import Optional, List, Literal
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.entity import Entity
+from zavod.shed import enforcements
+from zavod.shed.gpt import DEFAULT_MODEL, run_typed_text_prompt
+from zavod.stateful.review import (
+    Review,
+    assert_all_accepted,
+    request_review,
+    get_review,
+    model_hash,
+    html_to_text_hash,
+)
+
+NAME_XPATH = "//span[@class='treas-page-title']/text()"
+CONTENT_XPATH = "//div[@id='block-ofac-content']//div[@class='content']"
+DATE_XPATH = "//div[@id='block-ofac-content']//div[@class='field__item']/text()"
+# Traveling back to summer of 2016 (the first full pdf enforcement was issued on July 5, 2016)
+MAX_AGE_DAYS = (date.today() - date(2016, 7, 8)).days
+
+Schema = Literal["Person", "Company", "LegalEntity"]
+Status = Literal[
+    "Filed",
+    "Dismissed",
+    "Settled",
+    "Final judgement",
+    "Other",
+]
+
+# Settlement Agreement
+# Penalty Notice to an Individual
+# Finding of Violation
+# Sanctions Compliance Guidance
+# Issuance of Sanctions Regulations
+# Civil Monetary Penalty
+# Release of OFAC Enforcement Information
+# Release​ of Civil Penalties Information
+# Recent OFAC Actions
```

Comment:
should this be worked into Status options? Do we still need it? It looks interesting but if we wanna keep it we might wanna have a comment explaining what it means.